sdk/digitaltwins/Azure.DigitalTwins.Core/src/Queries/QueryBuilders/Linq/DigitalTwinsFunctions.cs

```diff
@@ -0,0 +1,133 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Azure.DigitalTwins.Core.QueryBuilder
+{
+    /// <summary>
+    /// DigitalTwins Query Language <see href="https://docs.microsoft.com/azure/digital-twins/reference-query-functions">Functions</see>.
+    /// </summary>
+    public static class DigitalTwinsFunctions
+    {
+        /// <summary>
+        /// Adds the <see href="https://docs.microsoft.com/azure/digital-twins/reference-query-functions#is_defined">IS_DEFINED</see> function to the condition statement of the query.
+        /// </summary>
+        /// <param name="value">The property that the query is looking for as defined.</param>
+        /// <returns>True if the specified property is defined within the scope of the query, false if otherwise.</returns>
+        public static bool IsDefined(object value) => throw new NotImplementedException();
```

Comment:
It'd be nice to add a helpful message in these like:
```suggestion
        public static bool IsDefined(object value) => throw new NotImplementedException($"The {nameof(IsDefined)} method is evaluated by the service and can only be used within {nameof(DigitalTwinsQueryBuilder)}.{nameof(DigitalTwinsQueryBuilder.Where)}.");
```